build/omni-nav.bundle.js

```diff
@@ -59,130 +64,35 @@
-.42-2.43a2.42 2.42 0 0 0-1.93-.74v-.51h3.2l9.22 11v-5.19a34.24 34.24 0 0 0-.17-4.16c-.11-.77-.7-1.07-2.15-1.14v-.51h5.26v.5c-1.21 0-1.78.37-1.88 1.14a34.22 34.22 0 0 0-.17 4.16v8.59zM185.51 17.8c0 2.39.17 2.54 2 2.67V21h-5.81v-.5c1.8-.13 2-.29 2-2.67V9.91c0-2.39-.17-2.54-2-2.67v-.51h5.78v.5c-1.8.13-2 .29-2 2.67v7.9zM195.24 21.11c-1.49-3.92-3.59-9.33-4.54-11.77-.7-1.82-1-2-2.32-2.1v-.51h5.32v.5l-.7.09c-.81.11-.94.33-.7 1 .57 1.62 2.26 5.94 3.86 10 1.14-3 3-8 3.51-9.42.42-1.25.2-1.53-.79-1.64l-.66-.07v-.46H203v.5c-1.47.15-1.66.31-2.48 2.24-.29.68-3 7.19-4.65 11.63h-.66zM205.9 9.91c0-2.39-.17-2.54-1.93-2.67v-.51h9.6c0 .37.13 2 .24 3.2l-.55.07a4 4 0 0 0-.79-2c-.35-.37-1.07-.53-2.41-.53h-1.67c-.61 0-.66 0-.66.68v5H210c1.88 0 2-.13 2.17-1.75h.55v4.34h-.55a2.35 2.35 0 0 0-.46-1.49 2.77 2.77 0 0 0-1.71-.28h-2.21v3.88c0 1.25.13 1.91.55 2.17a5 5 0 0 0 2.23.26c1.25 0 2.28-.11 2.76-.63a6.08 6.08 0 0 0 1.05-2.15l.55.07c-.11.61-.59 2.85-.77 3.44h-10.41v-.5c2-.13 2.15-.29 2.15-2.67V9.91zM220 17.8c0 2.39.17 2.54 2 2.67V21h-5.83v-.5c1.84-.13 2-.29 2-2.67V9.91c0-2.39-.17-2.54-1.93-2.67v-.51h5.57a6.4 6.4 0 0 1 3.46.75 3.19 3.19 0 0 1 1.45 2.8 4.09 4.09 0 0 1-2.89 3.83c.39.66 1.29 2.17 2 3.13a20.17 20.17 0 0 0 1.84 2.41 2.83 2.83 0 0 0 1.64 1.1v.42H229c-2.52-.07-3.29-.83-4.12-2-.68-1-1.6-2.61-2.21-3.64a1.4 1.4 0 0 0-1.38-.79H220v3.06zm0-3.73h1.27a3.24 3.24 0 0 0 2.19-.59 3.53 3.53 0 0 0 1.27-2.89 3.06 3.06 0 0 0-3.31-3.22 3.87 3.87 0 0 0-1.23.11c-.13 0-.2.15-.2.59v6zM230.55 17.12c.35 1 1.47 3.46 3.79 3.46a2.59 2.59 0 0 0 2.72-2.83c0-1.86-1.42-2.63-2.83-3.31-.72-.35-3.68-1.38-3.68-4 0-2.17 1.64-4 4.51-4a6.44 6.44 0 0 1 1.82.29 8.55 8.55 0 0 0 .83.24c.09.77.22 1.64.39 3l-.55.07c-.37-1.29-1-2.83-3-2.83a2.28 2.28 0 0 0-2.37 2.43c0 1.62 1.16 2.34 2.83 3.09 1.45.64 3.75 1.53 3.75 4.25 0 2.54-2.08 4.4-4.86 4.4a7 7 0 0 1-2.08-.33c-.55-.17-.94-.35-1.21-.46-.17-.57-.42-2.13-.64-3.31zM244.28 17.8c0 2.39.17 2.54 2 2.67V21h-5.78v-.5c1.8-.13 2-.29 2-2.67V9.91c0-2.39-.17-2.54-2-2.67v-.51h5.78v.5c-1.8.13-2 .29-2 2.67v7.9zM254.73 17.8c0 2.39.18 2.54 2.32 2.67V21h-6.4v-.5c2.06-.13 2.24-.29 2.24-2.67V7.46h-1c-2 0-2.54.29-2.87.7a5.81 5.81 0 0 0-.77 1.88h-.55c.09-1.34.22-2.74.26-3.86h.31a1 1 0 0 0 1.14.55h9a1.05 1.05 0 0 0 1-.55h.31c0 .92.13 2.52.24 3.79l-.56.03a4.26 4.26 0 0 0-.79-2c-.37-.42-1.05-.57-2.41-.57h-1.47V17.8zM267.95 17.8c0 2.39.17 2.54 2.28 2.67V21h-6.31v-.5c2-.13 2.19-.29 2.19-2.67V15.3a2.27 2.27 0 0 0-.48-1.31c-1-1.71-1.84-3.29-2.78-4.91s-1-1.69-2.3-1.84v-.51h5.28v.5l-1 .13c-.55.09-.7.28-.33 1 1 1.84 2 3.66 3.07 5.46.92-1.75 1.91-3.55 2.72-5.35.37-.79.2-1-.63-1.1l-.88-.11v-.53h4.82v.5c-1.47.13-1.58.42-2.5 1.86s-1.82 3-2.76 4.8a1.89 1.89 0 0 0-.37 1.18v2.73z\"/></g>';\n    var $svgLogo = '<svg class=\"chapman-logo\" xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 273.58 28.08\">' + defs + paths + '</svg>';\n    return $svgLogo;\n  };\n\n  // Public API\n  return {\n    build: build\n  };\n}();\n\nmodule.exports = OmniNav;\n\n//# sourceURL=webpack:///./js/OmniNav.js?");
```

Comment:
Should we be using eval use in Javascript, I thought it opened up the posibility for eval injections?